integration_test/src/test_runner/test_generator_util.rs

```diff
@@ -156,7 +156,7 @@ fn get_expected_data_type(sql_type: &Value) -> CDataType {
                 SqlDataType::EXT_W_CHAR => CDataType::WChar,
                 SqlDataType::EXT_W_VARCHAR => CDataType::WChar,
                 SqlDataType::EXT_W_LONG_VARCHAR => CDataType::WChar,
-                SqlDataType::EXT_GUID => CDataType::Guid,
```

Comment:
If we don't want this here, we should probably delete the line.

Reply:
I think we want it